### ISSUE TITLE
Line charts broke with TypeError: d is not a function

### ISSUE THREAD

**Issue by nborland27** [google/google-visualization-issues#2404]
This afternoon (2016-11-22) my LineChart application broke; and when I went to see if anything had changed, I saw that it broke even on google's example page:

https://developers.google.com/chart/interactive/docs/gallery/linechart

For each chart we see "loader.js:149 Uncaught TypeError: d is not a function"

I imagine that this problem has a lot of sites down that depend on this service.  Hopefully it can be corrected quickly.

**Comment by dlaliberte** [comment]
Yes, sorry about that. My bad! I broke all charts using the new loader with version 45 (or current). I updated for the new loader for the new version, but forgot a backward compatibility check. I repaired it quickly, but the old version has a cache lifetime of 1 hour.

So this should be fixed within an hour. I shortened the lifetime to 1 minute. If you can switch to using '44' instead of 'current' or '45' (or 'upcoming') that should fix it for you now.

**Comment by nborland27** [comment]
OK, thanks for the quick catch!
